src/Core/src/Handlers/Picker/PickerHandler.Android.cs

```diff
@@ -109,7 +110,7 @@ void OnClick(object? sender, EventArgs e)
 		{
 			if (_dialog == null && VirtualView != null)
 			{
-				using (var builder = new AppCompatAlertDialog.Builder(Context))
+				using (var builder = new MaterialAlertDialogBuilder(Context))
```

Comment:
Do we need to switch the control? I worry that for net9 this may cause issues with styling as none of the other dialogs are material. We got things like buttons and toolbars, but this full switch may need to wait for net10.

The single selection does exist:
https://developer.android.com/reference/androidx/appcompat/app/AlertDialog.Builder#setSingleChoiceItems(java.lang.CharSequence[],int,android.content.DialogInterface.OnClickListener)